Documenting team change process (#13240)
PostHog/posthog.com: contents/handbook/company/team-changes.md

- Are there considerations the proposer isn't aware of that could impact the decision we make? Please share them and suggest solutions. Often these relate to feature ownership. 

- Is there a better or alternative solution? Disagreeing with a proposal is fine, but it's always best to propose a solution than to just disagree without an alternative. If you think no change is necessary, explain why.

- Be direct and clear about how strongly you feel. If you're strongly against a change, explain why and make that clear. Likewise, if you're unsure about a change, but don't feel strongly, articulate that. Consensus is not our goal and decisions being blocked by people who are less invested in the outcome will slow us down and lead to worse decision-making.

</CalloutBox>

### 3. Share the final decision in Slack

The final decision should always be made by the relevant member(s) of the Blitzscale Team in a timely fashion.

Once made, they should share their decision in #tell-posthog-anything and the relevant team channels, alongside a short summary of why we're making that change.

## FAQ

### What if I want to move teams?

> Individuals wishing to change teams should speak directly with their line manager. This process exists purely for making larger changes to existing teams, or forming new ones, that impact multiple people.

Reviewer: this sentence reads weird

> Individuals wishing to change teams should speak directly with their line manager. This process exists purely for making larger changes to existing teams, or forming new ones, that impact multiple people.

Author: ```suggestion
Talk to your line manager. This process exists purely for making larger changes to existing teams, or forming new ones, that impact multiple people.
```
